src/content/reference/k8s-metadata/labels/_index.md

```diff
@@ -0,0 +1,147 @@
+
+### giantswarm.io/logging
+
+With this label on the Cluster resource, logging via Loki can be disabled. The default value `true` means that logging is enabled. Setting it to `false` disables logging for the entire cluster.
+
+### giantswarm.io/machine-pool
+
+Associates the resource or the node with a node pool, using the name of the node pool as the value.
+
+### giantswarm.io/managed-by
+
+- TODO Found on various resources in MC and WC. Value differs.
+    - E. g. `rbac-operator` on RBAC resources indicates that rbac-operator created these and will reconcile/delete them.
+    - Value `flux` on an App resource indicates that the App is reconciled by Flux.
+    - Name of a bundle on an App resource indicates that the App has been installed by a bundle.
+
+### giantswarm.io/monitoring
+
+This label set on the Cluster resource with the value `false` can be used to disable the gathering of Prometheus metrics for the entire cluster, and id prevents deployment of Prometheus for this cluster.
+
+On other resources, the label serves as a discovery mechanism for metrics scraping. We're replacing this mechanism by service monitors.
+
+### giantswarm.io/organization
+
+An organization is a key concept in the Giant Swarm platform, used to isolate [tenants]({{< relref "/vintage/platform-overview/multi-tenancy" >}}). This label is used on a variety of resources to associate them with one organization. The value is the name of the organization.
+
+### giantswarm.io/prevent-deletion
+
+Can be set on certain resource types to prevent deletion. See [Prevent accidental deletion of resources]({{< relref "/vintage/advanced/app-platform/deletion-prevention" >}}).
+
+### giantswarm.io/service-type
+
+- TODO Found on various resources in management and workload clusters.
+- [`fmt`](https://github.com/giantswarm/fmt/blob/278eb4b3318c454e50f24413e7ef2250159f28d6/kubernetes/annotations_and_labels.md?plain=1#L49) has an explanation.
+Value is mostly `managed`
+- also found occurrence `manual` in an [ops recipe](https://github.com/giantswarm/giantswarm/blob/0d16eb4ebb0440608bb1bfd0636d34afa6352cc6/content/docs/support-and-ops/ops-recipes/cilium-rate-limit-issue.md?plain=1#L39).
+- Value `system` is also documented.
+- Example use: [PrometheusRule](https://github.com/giantswarm/prometheus-rules/blob/d440f5cc724d9ad2fe4c7f85c9f0b54090a6858e/helm/prometheus-rules/templates/recording-rules/gs-managed-app-deployment-status.rules.yml#L16) - Here it seems to be used to differentiate workloads managed by Giant Swarm from other workloads.
+
+### giantswarm.io/service-priority
+
+Used on cluster-related resources to indicate the [service priority]({{< relref "/vintage/advanced/cluster-management/labelling-workload-clusters#service-priority" >}}), which means the relative importance in general, of the cluster.
+
+### policy.giantswarm.io/psp-status
+
+- TODO Found on App, Chart, and Cluster resources, MC and WC. On clusters with PSS the value is `disabled`.
```

Comment:
@giantswarm/team-shield is still used?

Reply:
It is used internally by a few components related to PSP deprecation/migration. This label can be removed after CAPI migration is complete. But the label itself isn't a customer API

Reply:
ok then I suggest to remove it